File: .old2/code/LightScript/examples/js/prime.js
function isPrime(x) {
    for(d = 2; d < x; ++d) {
        if((x % d) === 0 ) {
            return false;
        }
    }
    return true;
}

for(i = 9999990; i< 10000020; ++i){
    if(isPrime(i)) {
        print(i);
    }
}

File: .attic2/code/bench/kahlua/prime.lua
function isPrime(x) 
    for d = 2, x - 1 do
        if (x % d) == 0 then
            return false
        end
    end
    return true;
end

for i = 9999990, 10000019 do
    if isPrime(i) then
        print(i)
    end
end
